### PR TITLE
fixup emoji add errors

### PR DESCRIPTION
**Pull request by mmaxim**
cc @keybase/hotpotatosquad

### CODE REVIEW

File: go/chat/emojisource.go [hunk 1]
```diff
@@ -165,21 +207,25 @@ func (s *DevConvEmojiSource) normalizeShortName(shortName string) string {
...
+	}
+	if len(shortName) > maxShortNameLength {
+		return NewEmojiValidationErrorJustError(fmt.Errorf("short name is too long, must be less than %d",
+			maxShortNameLength))
 	}
-	if len(shortName) > maxShortNameLength || len(shortName) < minShortNameLength {
-		return fmt.Errorf("short name (length %d) not within bounds %d,%d",
-			len(shortName), minShortNameLength, maxShortNameLength)
+	if len(shortName) < minShortNameLength {
+		return NewEmojiValidationErrorJustError(fmt.Errorf("short name is too short, must be greater than %d",
```
Comment (songgao) [inline go/chat/emojisource.go:219]: This is more clear from the beginning but might still be too long for 5s: 

image

Comment (songgao) [reply go/chat/emojisource.go:219]: or maybe we could just make the error display two lines on mobile?

File: go/chat/emojisource.go [hunk 2]
```diff
@@ -32,6 +33,47 @@ const (
...
+
+func NewEmojiValidationErrorSimple(err error, display string) *EmojiValidationError {
+	return &EmojiValidationError{
+		Underlying: err,
+		CLIDisplay: display,
+		UIDisplay:  display,
+	}
+}
+
+func NewEmojiValidationErrorJustError(err error) *EmojiValidationError {
```
Comment (joshblum) [inline go/chat/emojisource.go:69]: `JustError` -> `Raw`?

**Comment by adamjspooner** [comment]
I can take a pass at making the error messages shorter if you want to get this in.

**Pull request merged**